cmd/thanos/tools_bucket.go

```diff
@@ -229,6 +230,8 @@ func (tbc *bucketReplicateConfig) registerBucketReplicateFlag(cmd extkingpin.Fla
 
 	cmd.Flag("single-run", "Run replication only one time, then exit.").Default("false").BoolVar(&tbc.singleRun)
 
+	cmd.Flag("mark-after", "Mark replicated blocks for deletion in source bucket.").Default("false").BoolVar(&tbc.markAfter)
```

Comment:
I don't like the flag name. Can we just call it `mark-deletion` or `delete` something like that. `mark-after` is confusing because Thanos has 3 types of markers

Reply:
Okay, i agree, `mark-deletion` would be more informative.